versioned_docs/version-v4.0.0/node/troubleshooting.md

```
@@ -0,0 +1,662 @@
+</summary>
+
+<Tabs queryString="retrieve-logs">
+<TabItem value="docker" label="Docker">
+
+1. Connect to your machine and execute the command `docker ps`. This will provide you with a list of Docker containers you are currently running. Among them, locate the container with the label **europe-west3-docker.pkg.dev/hoprassociation/docker-images/hoprd:stable** and note the **container ID**.
+
+2. Get the logs from the docker container using the following command: `docker logs -t <Your_Node_Container_ID> >> <File_name.log>`. Replace **\<Your_Node_Container_ID\>** with your docker container ID. Replace **\<File_name.log\>** with your container ID and **\<File_name.log\>** with your chosen file name. After executing the command, wait until it finishes writing the logs to the file.
+
+    **Example:**
+
+    ```md
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)